Home/Home.html

```diff
@@ -1,13 +1,165 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
     <script src="Home_js.js"></script>
     <link rel="stylesheet" href="Home_style.css">
 </head>
+
 <body>
-    
+    <!-- Hambergur menu -->
+    <div id="menu">
+        <div id="menu_items">
```

Comment:
- [ ] I want to kindly suggest that you make use of the semantic `header` tag to display the headers of the project. It is important to use semantic tag for search engine optimization and it assists screen readers and other assistive technologies👍 ✨